__tests__/app.test.js

```diff
@@ -3,6 +3,7 @@ const endpointsJson = require("../endpoints.json")
 // app and request
 const app = require("../app")
 const request = require("supertest")
+require("jest-sorted")
```

Comment:
You shouldn't need to require in jest sorted - we actually need to add the set up code to our package.json - you can find this in the jest sorted docs!